src/nrf_802154_core.c

```diff
@@ -2481,6 +2482,7 @@ bool nrf_802154_core_transmit(nrf_802154_term_t              term_lvl,
                 state_set(cca ? RADIO_STATE_CCA_TX : RADIO_STATE_TX);
                 mp_tx_data = p_data;
 
+                // coverity[check_return]
```

Comment:
I was thinking if instead of suppressing coverity warning in a comment, would it be possible to simply add `(void)result;` before `result = true;` to the `else` block below?

Reply:
No, it doesn't work here. I have made such attempts before:
- to insert `(void)result;` right after `result = tx_init( . . );` 
- to insert it in `else { (void)result; result = true; }`.
For some unknown reason, neither of the above worked.